TranslatorReasonerAPI.yaml

```diff
@@ -879,6 +886,7 @@ components:
             Node attributes provided by this TRAPI web service.
           items:
             $ref: '#/components/schemas/MetaAttribute'
+          nullable: true
```

Comment:
I would prefer a sensible default to `null`:
```suggestion
          default: []
```

Reply:
Per WG discussion, the empty array here actually means a different thing than null.